spec/chromedriver_updater_spec.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.describe Geordi::ChromedriverUpdater do
+  describe '#run' do
+    context 'exit_on_failure=true (default)' do
+      it 'exits the command with status code 1 if an error occurs' do
+        allow(subject).to receive(:determine_chrome_version).and_raise(Geordi::ChromedriverUpdater::ProcessingError, 'Could not determine the version of Google Chrome.')
+        expect(Geordi::Interaction).to receive(:fail).with('Could not determine the version of Google Chrome.')
+
+        subject.run({})
+      end
+    end
+
+    context 'exit_on_failure=false' do
+      it 'prints an interaction note if an error occurs' do
+        allow(subject).to receive(:determine_chrome_version).and_raise(Geordi::ChromedriverUpdater::ProcessingError, 'Could not determine the version of Google Chrome.')
+        expect(Geordi::Interaction).to receive(:note).with('Could not determine the version of Google Chrome.')
+
+        subject.run(exit_on_failure:false)
```

Comment:
I have no clue how these tests could be green with the present code, as method names are twisted.

Reply:
Good question. I might have done some refactoring after writing the tests and not forgot to run them again. 
And it seems our github actions test setup only runs feature tests. We need to change that.